src/plots/gl3d/scene.js

```diff
@@ -172,6 +171,16 @@ function initializeGLPlot(scene, fullLayout, canvas, gl) {
         showNoWebGlMsg(scene);
     }
 
+    var relayoutCallback = function(scene) {
+        var update = {};
+        update[scene.id] = getLayoutCamera(scene.camera);
+        scene.saveCamera(scene.graphDiv.layout);
```

Comment:
This breaks if `scene.graphDiv.layout` is not fully set.

For example, using the [`gl3d_errorbars_zx`](https://github.com/plotly/plotly.js/blob/master/test/image/mocks/gl3d_errorbars_zx.json) mock - which doesn't completly set [`scene.camera`](https://github.com/plotly/plotly.js/blob/123e453e7a93afced649bd8b034a0eb65c42fca3/test/image/mocks/gl3d_errorbars_zx.json#L48-L53) - lead on every mouse up, or scroll to:

![image](https://cloud.githubusercontent.com/assets/6675409/15057126/3f3e6dc8-12e3-11e6-95a1-b6c359d0eeb2.png)

The [`saveCamera`](https://github.com/plotly/plotly.js/blob/123e453e7a93afced649bd8b034a0eb65c42fca3/src/plots/gl3d/scene.js#L615) method needs to be patched.